pkg/helpers/version/version.go

```diff
@@ -44,66 +44,6 @@ func GetVersionBundle(version string) (VersionBundle, error) {
 
 	// predefined bundle version
 	// TODO: automated version tracking
-	versionBundleList["0.5.0"] = VersionBundle{
```

Comment:
so this means we only support n-3 versions?

Reply:
I think so, we can not support earlier version since the change on operator manifests.